src/object.d

```diff
@@ -2377,6 +2377,8 @@ class Throwable : Object
     {
         this.msg = msg;
         this.nextInChain = nextInChain;
+        if (nextInChain && nextInChain._refcount)
+            ++nextInChain._refcount;
```

Comment:
I don't believe the code coverage checked which claims this line is not covered - if I remove it the tests fail.